### PR TITLE
Bump heroku stack to Heroku:18

### PR DESCRIPTION
**Pull request by swistaczek**
Hello 👋 ,
This PR updates base Heroku (heroku-build) stack from tag 16 to latest supported tag 18. I removed release as discussed in https://github.com/gliderlabs/herokuish/pull/436. /cc @michaelshobbs

### CODE REVIEW

File: buildpacks/buildpack-multi/tests/multi/Gemfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
 source 'http://rubygems.org'
-ruby '2.2.4'
+ruby '2.6.1'
 gem 'sinatra'
 gem 'rack'
```
Comment (swistaczek) [inline buildpacks/buildpack-multi/tests/multi/Gemfile]: Ruby 2.2.4 is no longer supported version for Heroku:18.

File: buildpacks/buildpack-ruby/tests/ruby-sinatra/Gemfile.lock [hunk 2]
```diff
@@ -7,7 +7,7 @@ GEM
       rack
     rack-test (0.6.3)
       rack (>= 1.0)
-    rake (12.0.0)
+    rake (12.3.2)
```
Comment (swistaczek) [inline buildpacks/buildpack-ruby/tests/ruby-sinatra/Gemfile.lock]: Minor change, use system available rake instead of installing old one (outdated)

**Comment by swistaczek** [comment]
Hey guys, any chance of merging this into upstream :)?

**Comment by josegonzalez** [comment]
Yes but it needs to be reviewed, so you'll need to be patient. The maintainers have lives outside of open source ;)

@michaelshobbs thoughts on this one? It seems fairly straightforward, though we'll need to release a minor to match it.

**Comment by michaelshobbs** [comment]
Thanks for the PR! I should have time on Wednesday to look this over more thoroughly.

**Pull request merged**